src/markdown.cpp

```diff
@@ -937,6 +982,22 @@ int Markdown::processLink(const char *data,int,int size)
       TRACE_RESULT(0);
       return 0;
     }
+    // check if there is text before '![' to know if the image needs to be inlined
```

Comment:
I don't get this, do you imply here that only when the `![` is not at the beginning of of line it will be in-lined? What happens when you have it in  block of text and due to layouting the image would start at the beginning of a line.?

Reply:
I've done some updates in 715298173ac90d9d60516a1a711c575f688a6a54.

So at least 2 newline before and after image tag, of course whitespaces are ignored when checking...